baselines/dom.generated.d.ts

```diff
@@ -1819,9 +1819,12 @@ declare var CryptoKeyPair: {
 };
 
 interface CSS {
```

Comment:
I'd expect that something like the below to be the generated output

```ts

interface CSS {
    supports(property: string, value?: string): boolean;
    escape(value: string): string;
}

declare var CSS: CSS;
```

However I cannot seem to make it like that.

Reply:
the generation script is not handling added static properties correctly.. i think i have a fix for that.